.travis.yml

```diff
@@ -15,11 +15,11 @@ install:
 - gem update --system
 - gem install sass
 - gem install ruby_dep -v 1.3.1
-- gem install jekyll -v 3.2.1
+- gem install jekyll -v 4.0.0
 before_script:
 - if [ $TRAVIS_PULL_REQUEST = 'false' ]; then git checkout -qf $TRAVIS_BRANCH; fi
 script:
-- if [ $TRAVIS_SCALA_VERSION = '2.12.8' ];
+- if [ $TRAVIS_SCALA_VERSION = '2.12.11' ];
```

Comment:
We should be able to build this on `2.13` now that we have mdoc

Reply:
Fixed

Reply:
Hmm, it is now failing for all versions except 2.13.2. Don't understand why yet.